sites/all/modules/publizon/publizon.module

```diff
@@ -584,3 +584,24 @@ function publizon_client_library_load($library_id) {
 
   return $libraries[$library_id];
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function publizon_form_alter(&$form, &$form_state, $form_id) {
+  // Set default value on support_id webform field.
+  if (isset($form['submitted']['support_id'])) {
+    global $user;
+    if ($user->uid > 0) {
```

Comment:
There is an api function `user_is_logged_in` see https://api.drupal.org/api/drupal/modules!user!user.module/function/user_is_logged_in/7.x

Reply:
Code changed to use `user_is_logged_in`.